AutoNode/validator.py

```diff
@@ -0,0 +1,170 @@
+    print(f"{Typgpy.OKBLUE}Verifying Balance...{Typgpy.ENDC}")
+    # Check validator amount +1 for gas fees.
+    if not check_min_bal_on_s0(validator_config['validator-addr'], validator_config['amount'] + 1, node_config['endpoint']):
+        print(f"{Typgpy.FAIL}Cannot create validator, {validator_config['validator-addr']} "
+              f"does not have sufficient funds.{Typgpy.ENDC}")
+        raise RuntimeError("Create Validator Error")
+    else:
+        print(f"{Typgpy.OKGREEN}Address: {validator_config['validator-addr']} has enough funds{Typgpy.ENDC}")
+    _verify_node_sync()
+    print(f"{Typgpy.OKBLUE}Sending create validator transaction...{Typgpy.ENDC}")
+    _send_create_validator_tx()
+
+
+def _send_create_validator_tx():
+    wait_for_node_response(node_config['endpoint'], verbose=True)
+    try:
+        response = cli.single_call(f'hmy --node={node_config["endpoint"]} staking create-validator '
+                                   f'--validator-addr {validator_config["validator-addr"]} '
+                                   f'--name "{validator_config["name"]}" '
+                                   f'--identity "{validator_config["identity"]}" '
+                                   f'--website "{validator_config["website"]}" '
+                                   f'--security-contact "{validator_config["security-contact"]}" '
+                                   f'--details "{validator_config["details"]}" '
+                                   f'--rate {validator_config["rate"]} '
+                                   f'--max-rate {validator_config["max-rate"]} '
+                                   f'--max-change-rate {validator_config["max-change-rate"]} '
+                                   f'--min-self-delegation {validator_config["min-self-delegation"]} '
+                                   f'--max-total-delegation {validator_config["max-total-delegation"]} '
+                                   f'--amount {validator_config["amount"]} '
+                                   f'--bls-pubkeys {",".join(node_config["public-bls-keys"])} '
+                                   f'--passphrase-file "{saved_wallet_pass_path}" '
+                                   f'--bls-pubkeys-dir "{bls_key_dir}" ')
+        print(f"{Typgpy.OKBLUE}Created Validator!\n{Typgpy.OKGREEN}{response}{Typgpy.ENDC}")
+    except (json.decoder.JSONDecodeError, RuntimeError) as e:
+        print(f"{Typgpy.FAIL}Failed to edit validator!\n\tError: {e}{Typgpy.ENDC}")
+
+
+def _setup(recover_interaction):
+    wait_for_node_response(node_config['endpoint'], verbose=True)
+    all_val_address = get_all_validator_addresses(node_config['endpoint'])  # Check BLS key with validator if it exists
+    if validator_config['validator-addr'] in all_val_address:
+        print(f"{Typgpy.WARNING}{validator_config['validator-addr']} already in list of validators!{Typgpy.ENDC}")
+        if recover_interaction \
```

Comment:
`\` is not pythonic, please use a new named variable for the input_with_print

Reply:
done